src/Uno.UWP/UI/ViewManagement/ApplicationView.Android.cs

```diff
@@ -11,6 +12,33 @@ namespace Windows.UI.ViewManagement
 {
 	partial class ApplicationView
 	{
+		public bool IsScreenCaptureEnabled
+		{
+			get
+			{
+				if (!(ContextHelper.Current is Activity activity))
+				{
+					throw new InvalidOperationException($"{nameof(IsScreenCaptureEnabled)} API must be called when Activity is created");
+				}
+				return !activity.Window.Attributes.Flags.HasFlag(WindowManagerFlags.Secure);
+			}
+			set
+			{
+				if (!(ContextHelper.Current is Activity activity))
+				{
+					throw new InvalidOperationException($"{nameof(IsScreenCaptureEnabled)} API must be called when Activity is created");
+				}
+				if (value)
+				{
+					activity.Window.ClearFlags(WindowManagerFlags.Secure);
+				}
+				else
+				{
+					activity.Window.SetFlags(WindowManagerFlags.Secure, WindowManagerFlags.Secure);
```

Comment:
Even if usually we have only one activity in the application lifetime, I'm wondering if we should track new activities (with the `BaseActivity.CurrentChanged` static event for instance) and make sure to re-apply this flag on each new activity? Any thoughts @jeromelaban ?

Reply:
If we potentially plan to add multi-window (multiple application views), this would probably matter - so to apply the flag to the activity that is associated with the current ApplicationView. However, this should not happen otherwise, or can it?

Reply:
At this point, the feature as it is is enough. If it happens that multi-activity break, we'll adjust at that time.